in.go

```diff
@@ -11,6 +11,16 @@ import (
 
 // Get (business logic)
 func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResponse, error) {
+	if request.Params.SkipDownload != "" {
```

Comment:
Shouldn't we check if this is `true`?

Reply:
Afaik, Concourse passes booleans in as strings so it's up to us to parse it into a boolean, and few lines further down we check whether it parses to true. Or did I misunderstand you?

Reply:
My bad, I misread.